Tasks/PackerBuildV0/Tests/L0.ts

```diff
@@ -128,7 +128,7 @@ describe('PackerBuild Suite', function() {
                 assert(tr.invokedToolCount == 4, 'should have invoked tool four times. actual: ' + tr.invokedToolCount);
                 assert(tr.stderr.length == 0 || tr.errorIssues.length, 'should not have written to stderr');
                 assert(tr.succeeded, 'task should have succeeded');
-                assert(tr.stdout.indexOf("##vso[task.setvariable variable=imageUri;issecret=false;]https://bishalpackerimages.blob.core.windows.net/system/Microsoft.Compute/Images/packer/packer-osDisk.e2e08a75-2d73-49ad-97c2-77f8070b65f5.vhd") != -1, "image uri output variable not set");
+                assert(tr.stdout.indexOf("##vso[task.setvariable variable=imageUri;isOutput=false;issecret=false;]https://bishalpackerimages.blob.core.windows.net/system/Microsoft.Compute/Images/packer/packer-osDisk.e2e08a75-2d73-49ad-97c2-77f8070b65f5.vhd") != -1, "image uri output variable not set");
```

Comment:
what does `isOutput` do?

Reply:
isOutput=true means variable is read only and can only be set once.Any attempt of setting twice this would result into errors.isOutput=false is set as default. Reference  https://github.com/microsoft/azure-pipelines-yaml/blob/master/design/readonly-variables.md#solution-make-variables-readonly

Reply:
@bishal-pdMSFT